lib/phonebook/ui/pages/association_editor_page/association_editor_page.dart

```diff
@@ -0,0 +1,334 @@
+import 'package:myecl/phonebook/providers/association_list_provider.dart';
+import 'package:myecl/phonebook/providers/association_member_list_provider.dart';
+import 'package:myecl/phonebook/providers/association_member_sorted_list_provider.dart';
+import 'package:myecl/phonebook/providers/association_picture_provider.dart';
+import 'package:myecl/phonebook/providers/association_provider.dart';
+import 'package:myecl/phonebook/providers/complete_member_provider.dart';
+import 'package:myecl/phonebook/providers/membership_provider.dart';
+import 'package:myecl/phonebook/providers/roles_tags_provider.dart';
+import 'package:myecl/phonebook/router.dart';
+import 'package:myecl/phonebook/tools/constants.dart';
+import 'package:myecl/phonebook/ui/components/kinds_bar.dart';
+import 'package:myecl/phonebook/ui/phonebook.dart';
+import 'package:myecl/phonebook/ui/pages/association_editor_page/member_editable_card.dart';
+import 'package:myecl/tools/constants.dart';
+import 'package:myecl/tools/functions.dart';
+import 'package:myecl/tools/token_expire_wrapper.dart';
+import 'package:myecl/tools/ui/builders/async_child.dart';
+import 'package:myecl/tools/ui/builders/waiting_button.dart';
+import 'package:myecl/tools/ui/layouts/add_edit_button_layout.dart';
+import 'package:myecl/tools/ui/layouts/refresher.dart';
+import 'package:qlevar_router/qlevar_router.dart';
+
+class AssociationEditorPage extends HookConsumerWidget {
+  final scrollKey = GlobalKey();
+  AssociationEditorPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final key = GlobalKey<FormState>();
+    final association = ref.watch(associationProvider);
+    final associationNotifier = ref.watch(asyncAssociationProvider.notifier);
+    final associationMemberListNotifier =
+        ref.watch(associationMemberListProvider.notifier);
+    final associationMemberList = ref.watch(associationMemberListProvider);
+    final associationMemberSortedList =
+        ref.watch(associationMemberSortedListProvider);
+    final associationPictureNotifier =
+        ref.watch(associationPictureProvider.notifier);
+    final associationListNotifier = ref.watch(associationListProvider.notifier);
+    final kind = ref.watch(associationKindProvider);
+    final name = useTextEditingController(text: association.name);
+    final description = useTextEditingController(text: association.description);
+    final rolesTagsNotifier = ref.watch(rolesTagsProvider.notifier);
+    final membershipNotifier = ref.watch(membershipProvider.notifier);
+    final completeMemberNotifier = ref.watch(completeMemberProvider.notifier);
+    void displayToastWithContext(TypeMsg type, String msg) {
+      displayToast(context, type, msg);
+    }
+
+    return PhonebookTemplate(
+        child: Refresher(
+      onRefresh: () async {
+        await associationMemberListNotifier.loadMembers(
+          association.id,
+          association.mandateYear.toString(),
+        );
```

Comment:
Shouldn't we refresh the association's information too ?

Reply:
fixed